Comp2023/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -371,8 +375,8 @@ public void driveWithPathFollowerInit(PathPlannerTrajectory trajectory, boolean
     DataLogManager.log(getSubsystem( ) + ": DTR states: " + trajStates.size( ) + " dur: " + m_trajectory.getTotalTimeSeconds( ));
 
     // This initializes the odometry (where we are)
-    if (useInitialPose)
-      resetOdometry(m_trajectory.getInitialHolonomicPose( ));
+    //if (useInitialPose)
```

Comment:
I don't understand how this can work for back-to-back paths, unless the odometry is reset elsewhere.

Reply:
I added that back in, I had it commented out temporarily for the pathSequence that wasn't set back-to-back.